doc/operators_precedence.rst

```diff
@@ -0,0 +1,49 @@
+
+=========== ============== ======= ============= ===========
+Precedence  Operator       Type    Associativity Description
+=========== ============== ======= ============= ===========
+512         `.`            infix   Left          Get an attribute on a variable
+            `[`                                  Array access
+            `(`                                  Function call
+500         `-`            prefix  n/a
+            `+`
+300         `|`            infix   Left          Filter call
```

Comment:
is this describing the current precedence (in the 3.x version) or the future precedence defined by a PrecedenceChange ?

Reply:
The future precedence... which is what you "get" when you use parentheses (which is required to avoid deprecation notices). Maybe we should have both the current and the new precedence here? I didn't do it to avoid having too much information.

Reply:
when you use parentheses, the precedence of the operator does not matter anymore, as the parenthesis win anyway.

I think that the documentation in the 3.x branch needs to document the existing precedence as this is how parsing happens (we might add an info about the planned precedence change though)

Reply:
Updated